certs/provider.go

```diff
@@ -54,12 +54,6 @@ func (c *CertificateSummary) DomainNames() []string {
 	return c.domainNames
 }
 
-// ChainSize returns the number of intermediate certificates
-// in the chain
-func (c *CertificateSummary) ChainSize() int {
-	return len(c.chain.Subjects())
```

Comment:
This was removed because of this `staticcheck` error:

```
Error: certs/provider.go:60:[13](https://github.com/zalando-incubator/kube-ingress-aws-controller/runs/5848683312?check_suite_focus=true#step:8:13): SA1019: c.chain.Subjects is deprecated: if s was returned by SystemCertPool, Subjects will not include the system roots. (staticcheck)
	return len(c.chain.Subjects())
	           ^
make: *** [Makefile:27: lint] Error 1

```

The method was only used in tests so it can be safely dropped.